src/router/index.js

```diff
@@ -145,6 +145,12 @@ const routes = [
     component: () => import("../components/ListOrgs.vue"),
     meta: {pageTitle: "List organizations", requireAdmin: true}
   },
+  { 
+    path: "/list-users/:orgType/:orgId",
+    name: "ListUsers",
+    component: () => import("../components/ListUsers.vue"),
+    meta: {pageTitle: "List users", requireAdmin: true}
+  },
```

Comment:
I was never able to get this params implementation to work, I spent a long time trying it yesterday. 
I tried again today and now even after reverting the changes, my branch is still broken. so that's fun.